manipulation/src/main/java/org/jboss/gm/manipulation/ManipulationPlugin.java

```diff
@@ -50,6 +50,10 @@ public void apply(Project project) {
         final ManipulationModel alignmentModel = ManipulationIO.readManipulationModel(project.getRootDir());
         final ManipulationModel correspondingModule = alignmentModel.findCorrespondingChild(project);
 
+        if (System.getProperty("gmeFunctionalTest") != null) {
+            ConfigCache.getOrCreate(Configuration.class).reload();
+        }
+
         // we need to change the project version early so various tasks that ready early and create other vars based on it
```

Comment:
If we are running functional tests, trigger a reload.

Reply:
:+1: